edalize/templates/symbiflow/symbiflow-vpr-makefile.j2

```diff
@@ -0,0 +1,58 @@
+#Auto generated by Edalize
+
+TOP	:= {{ top }}
+VERILOG := {{ sources }}
+PARTNAME:= {{ partname }}
+DEVICE  := {{ part }}_test
+BITSTREAM_DEVICE := {{ bitstream_device }}
+SDC	:= {{ sdc }}
+PCF	:= {{ pcf }}
+XDC	:= {{ xdc }}
+BUILDDIR:= {{ builddir }}
+
+{% if options -%}
+VPR_OPTIONS = VPR_OPTIONS="{{ options }}"
+{%- endif %}
+
+{% if seed -%}
+VPR_SEED = VPR_SEED="--seed {{ seed }}"
+{%- endif %}
+
+{% if pcf -%}
+PCF_OPTS = -p ${PCF}
+{%- endif %}
+
+{% if xdc -%}
+XDC_OPTS = -x ${XDC}
+{%- endif %}
+
+{% if sdc -%}
+SDC_OPTS = -s ${SDC}
+{%- endif %}
+
+all: ${TOP}.bit
+
+${BUILDDIR}:
+	mkdir ${BUILDDIR}
+
+${TOP}.eblif: | ${BUILDDIR}
+	cd ${BUILDDIR} && symbiflow_synth -t ${TOP} -v ${VERILOG} -d ${BITSTREAM_DEVICE} -p ${PARTNAME} ${XDC_OPTS}
```

Comment:
I'm not super excited about calling the wrapper script instead of calling yosys directly. The wrapper only accepts verilog and xdc files, so we can't add tcl, vhdl, systemverilog etc and we can't pass yosys_synth_options. I'm ok with using the wrapper to begin with if we intend to fix it later on.

Reply:
Yes, we intend to fix it later on.